### PR TITLE
feature/restructure_and_configs

### PR DESCRIPTION
**Pull request by chrstrom**
LESSGOO

### CODE REVIEW

File: config/cfg.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from cmath import pi
+
+N = 513
+
+Fs = 1e6
+Ts = 1.0 / Fs
+
+A = 1
+phi = pi / 8.0
+sigma_noise = 1.0
```
Comment (oyssolbo) [inline config/cfg.py]: Shouldn't sigma_noise be calculated from SNR? We could therefore just have SNR in the config-file

**Review by oyssolbo — CHANGES_REQUESTED** [review]
See comment

File: config/cfg.py (hunk 1 above)
Comment (chrstrom) [reply config/cfg.py]: sigma_noise is used in https://github.com/chrstrom/monkey_estimation/blob/b4bb576b833047172930aee5e73e14c23b74e940/scripts/Signals/Signals.py#L36 to generate the w[n] in x[n]. I just renamed it from "sigma" to "sigma_noise"

Comment (oyssolbo) [reply config/cfg.py]: Yes, but this value is not known beforehand. We only know it from the SNR

**Review by oyssolbo — CHANGES_REQUESTED** [review]

File: config/cfg.py (hunk 1 above)
Comment (chrstrom) [reply config/cfg.py]: Gotcha, fixed!

**Comment by chrstrom** [comment]
As can be seen in c97f477, I changed the file structure to make imports easier. (Things worked yesterday, then when I went to work on this now, the imports were broken again).

Sibling imports are dumb... anyways
Sadge

**Review by oyssolbo — APPROVED** [review]
lgtm

**Pull request merged**